packaged/bin/bxilog-check-order

```diff
@@ -1,14 +1,14 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # -*- coding: utf-8 -*-
-"""BXI Logging Parser."""
-###############################################################################
-# Author: Pierre Vignéras <pierre.vigneras@bull.net>
-###############################################################################
-# Copyright (C) 2012  Bull S. A. S.  -  All rights reserved
-# Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois
-# This is not Free or Open Source software.
-# Please contact Bull S. A. S. for details about its license.
-###############################################################################
+"""
+@author BXIHL <bxihl@atos.net>
```

Comment:
Do we remove the authors?
You could add a contact address but the author is the author...